--- a/vendor/nunomaduro/termwind/src/HtmlRenderer.php
+++ b/vendor/nunomaduro/termwind/src/HtmlRenderer.php
@@ -32,7 +32,7 @@
         }

         $html = '<?xml encoding="UTF-8">'.trim($html);
-        $dom->loadHTML($html, LIBXML_NOERROR | LIBXML_COMPACT | LIBXML_HTML_NODEFDTD | LIBXML_NOBLANKS | LIBXML_NOXMLDECL);
+        $dom->loadHTML($html, LIBXML_NOERROR | LIBXML_COMPACT | LIBXML_HTML_NODEFDTD /* | LIBXML_NOBLANKS */);

         /** @var DOMNode $body */
         $body = $dom->getElementsByTagName('body')->item(0);
